apps/style/curriculum/courses.scss

```diff
@@ -1,6 +1,7 @@
 @import "markdown";
 @import "pdf_printing";
 
+/* Copied in style/teacherDashboard/teacherDashboard.scss. Update there if anything is changed here. */
```

Comment:
Just curious - is there a way we could do this without having to manually update these separate files to maintain parity? Maybe importing the CSS from one to the other?

Reply:
Great suggestion. I was able to just import the two existing files